test/controllers/AddAdditionalMetadataControllerSpec.scala

```diff
@@ -590,6 +599,61 @@ class AddAdditionalMetadataControllerSpec extends FrontEndTestHelper {
     }
   }
 
+  "AddAdditionalMetadataController formFieldOverrides" should {
+    "override the field when the description is not empty" in {
```

Comment:
Maybe rename the test to make clear which field is being overridden: `override the 'descriptionClosed' field when the ...`